packages/odata-service-inquirer/src/translations/odata-service-inquirer.i18n.json

```
@@ -229,7 +221,16 @@
         "v2CatalogServiceNoAnnotations": "Error creating v2 catalog service object, annotations will not be available: {{-error}}"
     },
     "warnings": {
-        "largeMetadataDocument": "The metadata for this OData service is significantly large. It may take some time before this operation completes."
+        "largeMetadataDocument": "The metadata for this OData service is significantly large. It may take some time before this operation completes.",
+        "warningCertificateValidationDisabled": "User has disabled certificate validation",
+        "nonUIServiceTypeWarningMessage": "Please note that {{serviceType}} services, or not classified services, are not intended to be used for the generation of SAP Fiori UI applications",
+        "noServicesAvailable": "No services available for the selected system, see logs for further details.",
+        "noServicesAvailableForOdataVersion": "There are no V{{odataVersion}} OData services available from the selected system and the template you have chosen supports V{{odataVersion}} OData services only",
+        "noAnnotations": "No backend annotations associated with this service were retrieved and may result in an invalid application being created",
+        "collaborativeDraftMessage": "The selected service is draft enabled but does not support collaborative draft.",
+        "certificateErrors": "A certificate error is being returned when connecting to host: {{-url}}. Certificate error: {{error}}",
```

```suggestion
        "certificateErrors": "A certificate error occurred when connecting to the {{-url}} host. Certificate error: {{error}}.",
```

---

Okay, @IainSAP. Thanks for letting me know it is a log message. How about?
```suggestion
        "certificateErrors": "A certificate error occurred when connecting to the host: {{-url}}. Certificate error: {{error}}.",
```